### DISCUSSION TITLE
Pocket Base on Digital Ocean on-click installation and adding custom domain

### DISCUSSION THREAD

**Discussion by abdoKa** [pocketbase/pocketbase#1925] [Q&A]
Please how to enable SSL for my on-click installation for my sub-domain. using Caddy?

**Comment by ganigeorgiev** [comment]
Please note that the DO 1-Click app is not an officially maintained one.

You can find more info how to enable SSL/TLS with Caddy in https://caddyserver.com/docs/quick-starts/https
(_based on the 1-Click app description and the [packer script](https://github.com/scott/pocketbase-do-packer), you'll find the Caddy file in `/etc/caddy/Caddyfile`_).

**Reply by abdoKa** [reply]
Thank you @ganigeorgiev,

But I try to edit `/etc/caddy/Caddyfile`. but not result.
output:
:80

reverse_proxy :8090

my domain have HTTPS but my sub-domain based on main domain haven't please how to enable it in this case
